test/test.js

```diff
@@ -162,13 +149,15 @@ try {
     if (sentry.length) logOnDetails.sha_sentryfile = sentry;
 }
 catch (beef){
-    util.log("Cannae load the sentry. " + beef);
+    util.log("Cannot load the sentry. " + beef);
```

Comment:
Nah it's OK.  This script will soon be replaced by our robot overlords #147.

`Cannae` will return with basic examples though!